municipalities/beek-design-tokens/src/component/conduction/logo.tokens.json

```diff
@@ -0,0 +1,35 @@
+        },
+        "hover": {
+          "filter": {
+            "value": "opacity(80%)"
+          }
+        }
+      },
+      "footer": {
+        "inline-size": { "value": "91px" },
+        "block-size": { "value": "113px" },
+        "background-image": {}
+      },
+      "navbar": {
+        "inline-size": { "value": "91px" },
+        "block-size": { "value": "113px" },
+        "background-image": {
```

Comment:
The size of this logo is way out of proportions of the navbar
<img width="1151" height="183" alt="image" src="https://github.com/user-attachments/assets/be34c811-02bb-4d40-986f-e179789dcf25" />